dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -535,4 +536,19 @@ public Entry getAndCheckResource(User user, Long id) {
     public Entry getAndCheckResourceByAlias(String alias) {
         throw new UnsupportedOperationException("Use the TRS API for tools and workflows");
     }
+
+    @Override
+    public boolean canExamine(User user, Entry workflow) {
```

Comment:
I'd name the parameter entry instead of workflow, but I wouldn't change it unless you need to do another commit anyway.

Reply:
Yes, good call, I pushed an update that makes the naming much more accurate/consistent.